src/test/java/DropDown.java

```diff
@@ -0,0 +1,29 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.Select;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+public class DropDown extends BaseTest {
+    @Test
+    public void DropDownTest() {
+
+        driver.get("https://the-internet.herokuapp.com/dropdown");
+
+        Select select = new Select(driver.findElement(By.id("dropdown")));
```

Comment:
Please use "Select" class methods to select and verify that option is selected such as: selectByVisibleText(), selectByIndex(), getFirstSelectedElement() etc.